### PR TITLE
possible fix for #33872 - added additional mail placeholder

### PR DESCRIPTION
**Pull request by mglaubitz**
this is a possible fix for https://mantis.ilias.de/view.php?id=33872

please check it out - I hope I sorted the new lang entries correctly - and merge it to other branches as well (if you like)

### CODE REVIEW

File: Services/Mail/classes/class.ilAccountMail.php
Comment (mjansenDatabay) [inline Services/Mail/classes/class.ilAccountMail.php]:
```suggestion
```

File: Services/Mail/classes/class.ilAccountMail.php
Comment (mjansenDatabay) [inline Services/Mail/classes/class.ilAccountMail.php]:
```suggestion
```

File: lang/ilias_de.lang
Comment (mjansenDatabay) [inline lang/ilias_de.lang:7114]:
```suggestion
mail#:#reg_mail_user_activation_status#:#Aktivierungsstatus des Benutzerkontos
```

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]
Hi @mglaubitz,

thx for your contribution.

This LGTM except a few obsolete lines. Please provide the English translations, afterwards we could merge this if the maintainer of the `User` component is fine with this as well.

Best regards,
@mjansenDatabay

File: lang/ilias_de.lang
Comment (mglaubitz) [reply lang/ilias_de.lang:7114]: I have just added the English lang file. Committing correctly is not my strong side, it seems ;)

**Comment by mglaubitz** [comment]
I have committed the requested changes - thx for the feedback :)

**Review by mjansenDatabay — APPROVED** [review]

**Comment by mjansenDatabay** [comment]
Thx @mglaubitz !

@smeyer-ilias Any objections? I will cherry-pick this to the other relevant branches once you approved this.

**Review by smeyer-ilias — APPROVED** [review]
Yes, thanks. LGTM.

**Comment by mjansenDatabay** [comment]
@mglaubitz Please rebase the branch based on `release_7` and fix the conflicts. Afterwads I'll merge this and cherry-pick the changes to the other branches.

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]
@mglaubitz Please rebase the branch based on `release_7` and fix the conflicts. Afterwads I'll merge this and cherry-pick the changes to the other branches.

**Comment by chfsx** [comment]
As a technical board, we regularly check whether pull requests have been open for a long time.
Are there any updates on this? Please note that you can also close this request if you are not able or if you do not have the resources to look into it in detail.

At this point we would like to refer to the rules we all agreed on: https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/contributing.md#rules-for-maintainers-assigned-to-prs.

With kind regards!

**Comment by kergomard** [comment]
As a technical board, we regularly check whether pull requests have been open for a long time.

@mglaubitz could you please rebase this PR so that it can be merged?

Thank you very much and best!
@kergomard

**Comment by mjansenDatabay** [comment]
Hi @mglaubitz,

this PR has been lying around for quite a while. Please fix the conflicts. The maintainer (also @mjansenDatabay ) will merge/pick this afterwards.

Thanks and best,
@mjansenDatabay

**Comment by mjansenDatabay** [comment]
ILIAS 7 is no longer supported, so I'll close this PR.

**Pull request closed without merging**
